Wikipedia/Code/WebViewController.m

```diff
@@ -409,6 +417,11 @@ - (void)scrollToSection:(MWKSection*)section animated:(BOOL)animated {
     [self scrollToFragment:section.anchor animated:animated];
 }
 
+- (void)accessibilityCursorToSection:(MWKSection*)section {
+    NSString *js = [NSString stringWithFormat:@"focus_element = document.getElementById('%@'); other_element = document.body;  other_element.setAttribute('tabindex', 0); focus_element.setAttribute('tabindex', 0); other_element.focus(); focus_element.focus();", section.anchor];
```

Comment:
@bgerstle Ya I figure that would be a good follow-on for when we have time to make a sweep and do so for all inline JS strings :)